docs/user-guide/work-packages/set-change-dates/README.md

```diff
@@ -0,0 +1,138 @@
+---
+sidebar_navigation:
+  title: Set and change dates
+  priority: 890
+description: How to use the date picker to set and update start and finish of work packages
+keywords: date picker start finish dates change modify update relations work package
+---
+
+# Set and change start and finish dates of work packages
+
+| Feature                                                      | Documentation for                                         |
+| ------------------------------------------------------------ | --------------------------------------------------------- |
+| [Set start and finish dates](#set-start-and-finish-dates)    | How to set start and finish dates, and how to change them |
+| [Scheduling mode](#scheduling-mode)                          | Enabling manual scheduling and what it does               |
+| [Warning and information banners](#warning-and-information-banner) | What the different warning and information banners mean   |
```

Comment:
the anchor link is different to the headline, the link should be
 Information and warning banners
`[Warning and information banners](#information-and-warning-banner)`